### ISSUE TITLE
Migration from Simple File Manager not possible ?

### ISSUE THREAD

**Issue by Roos-AID** [gmansillo/simple#113]
The Simple File Manager is now replaced with Firedrive as I understood. But how can I migrate the existing categories and documents ? 
I could not find a solution yet. 
Any help welcomed. 

The Simple File Manager is a really good tool and I expect the same from Firedrive.

Thanks, Rob

**Comment by gmansillo** [comment]
Hi @Roos-AID,
Firedrive is the new name of Simple File Manager, the code behind is essentially the same.
Take a look to this similar request and please let me know if I can be of any further assistance: https://github.com/gmansillo/Firedrive/issues/110

**Comment by Roos-AID** [comment]
Thanks,  I have done the suggested changes. 
All files and categories have become visible in Firedrive. Only the Joomla menus needed to be changed. But then, testing it, I could not download any of the files.  
The files were moved from the SimpleFileManager.UPLOAD location to the Firedrive location correctly. But the files remained empty when downloading through firedrive.... 
Don't know what went wrong there.  
So, I changed all files in Firedrive and replaced with the correct versions. 

But most work was in the File and Categories and that has been migrated correctly.

Thanks for the scripts.
